### DISCUSSION TITLE
Semester 4 Week 5

### DISCUSSION THREAD

**Discussion by LPkkjHD** [SE-TINF22B6/Underwatch#211] [Announcements]
Hello and welcome to another weekly (yes we know, we missed the last week) update on the progress on our game Underwatch.

We will be kicking things off with the progress which has been made on the game.
Players are now able to hold and pick up new weapons. Weapons have been properly implemented featuring different fire rates, bullet damage as well as range.

https://github.com/SE-TINF22B6/Underwatch/assets/11832982/5864e6a5-fe08-4885-b75f-114100682692

Additionally we created many new assets (made them ready to be used in our game). Here the focus was on weapons as well as different enemy types.

We created some of the maps which we will be using in the final version of the game and we reworked the lighting system to allow maps to be outside under the sun as well as in the dungeon. This will be important for the story (lore) of the game (more on that in the next blog post).

https://github.com/SE-TINF22B6/Underwatch/assets/11832982/f49d633e-7e53-431d-bdcd-f1a61d294a73

Since we are now done with storycrafting the lore as well as theorycrafting items/guns as well as enemies we are now starting to add this information to the web page - resulting in the wiki becoming more and more useful by the day.
![image](https://github.com/SE-TINF22B6/Underwatch/assets/123720714/eb33017c-0fc0-4f9b-8dcd-60414a4c97d5)

Since we spend the majority of the time writing tests as well as creating a testing plan the list of new features has to end here. You're again more than welcome to read through our [Test Report](https://github.com/SE-TINF22B6/Underwatch/blob/main/documentation/Test%20Report.md)

**Comment by Maximon132** [comment]
Dear Underwatch,
I really like your new updates with the new maps and I really like the idea of having a lore in such a game. I am curious what the lore will be about and what else will come in the future.

**Comment by LenVej** [comment]
Dear Underwatch Team,
Your latest updates show some significant progress in testing, game progress and your wiki. I really like that you're planning out the lore and worldbuilding for your game and are now beginning to implement it. I'm looking forward to seeing what comes next!

Your Test Report demonstrates a commitment to ensuring the quality and stability of the game across its various components. The detailed breakdown of test strategies, plans, and cases reflects that you put time and effort into your testing.

Keep up the good work! 

Your CookHub Team

**Comment by maxschwinghammer** [comment]
Dear Underwatch,
exciting updates! The new weapon mechanics and assets sound great. The revamped lighting system adds depth, and I'm looking forward to more story details. Keep it up!
Your DHBWhub team

**Comment by Umbranecis** [comment]
Dear Underwatch,

Very Nice Progress you made there within the last two weeks. Im already excited to play your Game.
I love that you include videos so yweu can actually see what you did. The game looks great from what I can tell.
I also love your website design

Greetings from Team Plapy
